packages/@okta/vuepress-site/guides/sign-into-mobile-app/get-user-info/android/getinfo.md

```diff
@@ -14,4 +14,4 @@ client.getUserProfile(new RequestCallback<JSONObject, AuthorizationException>()
 });
 ```
 
-In `onSuccess` the user info returned is a `JSONObject` with the following [properties](https://developer.okta.com/docs/api/resources/oidc/#response-example-success-5).
\ No newline at end of file
+The `JSONObject` in `onSuccess` contains the user's info.
```

Comment:
@FeiChen-okta Can you add an example of pulling the user's email and name out of this object so it can be shown in the UI?